### ISSUE TITLE
New Use Case Submission

### ISSUE THREAD

**Issue by r-bartlett-gsa** [GSA/ai-experience-sharing-platform#30]
As a platform user, in order to contribute to the use case library, I would like to be able to to submit my AI use case.

MVP options:

- A form to submit contact details that would be populated in an email body that is sent to platform curator/admin upon form submission
- A single button that would initiated an email to a platform curator/admin and the email body would contain the email address of the logged-in user who clicked on the button
- Both options would display a confirmation/thank you page upon successful submission

**Comment by plummer-flex** [comment]
@cnelsonflexion Any guidance on where/how to implement this?

**Comment by cnelsonflexion** [comment]
Yes, have a look at the XD file. I just added a top level nav item and a CTA on the use case list views. I'm adding a content page that these will point to that will have some copy and the link that will fire the email

**Comment by cnelsonflexion** [comment]
Content for the submission screen:

Submit a Use Case

Do you have a project to share?

When you provide your use case, you help identify cutting edge solutions to common problems in government. Your submission will inform and inspire leaders, practitioners and researchers throughout the AI community. 

Here’s what we’ll need from you to get started: 

- A use case description of around 500 words
- A data set description, including relevant variables and analytical techniques
- The benefits produced by your use case
- Some contextual details (e.g., production status, business function, agency)
- Assurance of agency approval
- Keywords

Don’t worry if you aren’t sure about some of the details, you can leave any of the fields blank. Someone from the content team will reach out over email to confirm details and request more information. 

<email link>
